projects/demo-integrations/cypress/tests/kit/wrap-in-dialog/wrap-in-eager-dialog.cy.ts

```diff
@@ -0,0 +1,28 @@
+describe(`WrapInEagerDialog`, () => {
+    beforeEach(() => {
+        cy.viewport(`macbook-13`);
+    });
+
+    it(`should open dialog by click`, () => {
+        cy.tuiVisit(`/navigation/wrap-in-eager-dialog`);
+
+        cy.get(`tui-page-1-example button`).click();
+        cy.tuiWaitKitDialog();
+        cy.url().should(`include`, `/navigation/wrap-in-eager-dialog/path/to/dialog`);
+
+        cy.tuiClickOutside();
+        cy.url()
+            .should(`include`, `/navigation/wrap-in-eager-dialog`)
+            .should(`not.include`, `path/to/dialog`);
+    });
+
+    // Temporally broken
+    // Should work after fixing activeItemIndex = NaN;
```

Comment:
Did you create a separate PR? and then we can merge it before these

Reply:
Created PR for fixing activeItemIndex https://github.com/Tinkoff/taiga-ui/pull/3660